src/defs_version.h

```diff
@@ -53,6 +53,7 @@
     // In that case, this value is used
 
     // READ ABOVE CAREFULLY BEFORE CHANGING!
-    // USE 3 DIGITS VERSION ONLY /!\ _
-    #define MIXXX_VERSION "2.3.0-alpha-pre"
+    // USE 3 DIGITS VERSION ONLY !
+    // CHANGE IN ACTUAL VERSION FILE
+    #include "_version.h"
```

Comment:
Why prefix the file name with an underscore? Common convention?

Reply:
Nobody is supposed to include it, except the defs_version.h file. Underscore is a common prefix for private files in many languages.